src/kv/LockResolver.cc

```diff
@@ -253,11 +271,179 @@ void LockResolver::resolvePessimisticLock(Backoffer & bo, LockPtr lock, std::uno
     }
 }
 
+void LockResolver::resolveLockAsync(Backoffer & bo, LockPtr lock, TxnStatus & status)
+{
+    AsyncResolveDataPtr resolve_data{};
+    resolve_data = checkAllSecondaries(bo, lock, status);
+
+    status.commit_ts = resolve_data->commit_ts;
+
+    resolve_data->keys.push_back(lock->primary);
+    auto [keys_by_region, region_id] = cluster->region_cache->groupKeysByRegion(bo, resolve_data->keys);
```

Comment:
I think use [[maybe_unused]] is better?

Reply:
Seems attributes cannot work with structured bindings. Anyway, I have removed the unused variable.